src/workbench/SideBar.tsx

```diff
@@ -115,10 +115,12 @@ const SideBar = ({
     // A click on a tab when it's already selected should
     // reset any other parameters so we go back to the top
     // level.
-    setParams({
-      tab,
-    });
-  }, [tab, setParams]);
+    if (explore || reference) {
```

Comment:
This is quite subtle. I'm ok with merging but wonder whether it'd be possible to fix closer to the animation.